packages/cli/test/upgrade.test.js

```diff
@@ -0,0 +1,43 @@
+import { test } from 'tap'
+import { tmpdir } from 'os'
+import { execa } from 'execa'
+import { cp, readFile } from 'fs/promises'
+import { cliPath } from './helper.js'
+import { fileURLToPath } from 'url'
+import { dirname, join } from 'path'
+
+const pkg = JSON.parse(await readFile(join(dirname(fileURLToPath(import.meta.url)), '..', 'package.json'), 'utf8'))
+
+test('writes a config file', async (t) => {
+  const dest = join(tmpdir(), `test-cli-${process.pid}-1`)
+
+  await cp(
+    join(dirname(fileURLToPath(import.meta.url)), 'fixtures', 'v0.16.0.db.json'),
+    join(dest, 'platformatic.db.json'))
+
+  await execa('node', [cliPath, 'upgrade'], {
+    cwd: dest
+  })
+
+  const config = JSON.parse(await readFile(join(dest, 'platformatic.db.json'), 'utf8'))
+
+  t.equal(config.$schema, `https://platformatic.dev/schemas/v${pkg.version.replace('-dev', '')}/db`)
+})
+
+test('writes a config file with a config option', async (t) => {
+  const dest = join(tmpdir(), `test-cli-${process.pid}-1`)
```

Comment:
It's probably fine, but this uses the same destination as the previous test. Maybe change the `-1` or consider using `fs. mkdtemp()`.